src/MyHealth/SymptomLogContext.spec.tsx

```diff
@@ -102,7 +103,6 @@ describe("SymptomLogProvider", () => {
     it("fetch all daily check-in and log entries to serialize", async () => {
```

Comment:
This description seems outdated with what the body of the test is doing, should it be `get the log entries on load`